### ISSUE TITLE
Physical connection is not usable [xFFFFFFFF]

### ISSUE THREAD

**Issue by pivnicek** [keboola/db-extractor-mssql#49]
This should be A) retried, and B) a user exception
```
PDOException:SQLSTATE[08S01]: [Microsoft][ODBC Driver 17 for SQL Server]Session Provider: Physical connection is not usable [xFFFFFFFF]
```

**Comment by pivnicek** [comment]
This will need a test like https://github.com/keboola/db-extractor-common/pull/85

**Comment by pivnicek** [comment]
again today

**Comment by JakubMatejka** [comment]
+1 https://keboola.zendesk.com/agent/tickets/9646

**Comment by ondrejhlavacek** [comment]
again https://keboola.slack.com/archives/C09U3R1J4/p1547697837034500

**Comment by ondrejhlavacek** [comment]
again

https://connection.keboola.com/admin/utils/logs?file=2019/01/26/02/2019-01-26-02-05-33-5c4bb25db1ad1-exception

**Comment by ondrejhlavacek** [comment]
again 

https://connection.eu-central-1.keboola.com/admin/utils/logs?file=2019/01/29/08/2019-01-29-08-51-00-5c5005e45e599-exception

**Comment by kacurez** [comment]
again
https://keboola.slack.com/archives/C09U3R1J4/p1549250351002600

**Comment by tomasfejfar** [comment]
Also "Connection has been closed by peer" https://keboola.slack.com/archives/C09U3R1J4/p1551261006006400

**Comment by pivnicek** [comment]
:-( https://keboola.slack.com/archives/C09U3R1J4/p1551875710014700

**Comment by ujovlado** [comment]
Another err 500:

```
PDOException:SQLSTATE[08S01]: [Microsoft][ODBC Driver 17 for SQL Server]Session Provider: Connection has been closed by peer [xFFFFFFFF].
```

https://keboola.slack.com/archives/C09U3R1J4/p1553606409034200

**Comment by tomasfejfar** [comment]
`PDOException:SQLSTATE[08S01]: [Microsoft][ODBC Driver 17 for SQL Server]Session Provider: Connection has been closed by peer [xFFFFFFFF].`

https://keboola.slack.com/archives/C09U3R1J4/p1554106870011600

**Comment by tomasfejfar** [comment]
Happened multiple times today. I'll try to fix it - retry + user exception afterwards.

**Comment by JakubMatejka** [comment]
@tomasfejfar Jak to vypadá, dostal jsi se k tomu?

**Comment by tomasfejfar** [comment]
Myslel jsem, že to dám během ZD, ale nakonec mi to tady nějak zapadlo, sorry. Zapomněl jsem si to assignout :( 

Assignul jsem si to a někdy vyřeším nebo si to může vzít někdo jiný. Bojuju s db-ex-common...

**Comment by Halama** [comment]
Another `PDOException:SQLSTATE[08S01]: [Microsoft][ODBC Driver 17 for SQL Server]Session Provider: Connection has been closed by peer [xFFFFFFFF]`
https://keboola.slack.com/archives/C09U3R1J4/p1555983221021300

**Comment by pivnicek** [comment]
should by finally solved by 3.4.5
